tools/pyi/gen_pyi.py

```diff
@@ -11,6 +11,7 @@
 from typing import Sequence, List, Dict
 
 from tools.autograd.gen_python_functions import should_generate_py_binding, load_signatures, group_overloads
+from libfb.py import parutil
```

Comment:
@luciang this change adds libfb.py dependency that breaks pytorch build for open-source setups.
What is the origin of libfb?

Reply:
This PR is reverted now.